explorer/interpreter/resolve_unformed.h

```diff
@@ -43,49 +45,11 @@ class FlowFacts {
     FormedState formed_state;
   };
 
-  auto get_fact(Nonnull<const AstNode*> node) const
-      -> std::optional<Nonnull<const Fact*>> {
-    auto entry = facts_.find(node);
-    if (entry != facts_.end()) {
-      return &entry->second;
-    }
-    return std::nullopt;
-  }
-
-  void add_fact(Nonnull<const AstNode*> node, const FormedState state) {
-    CARBON_CHECK(!get_fact(node).has_value());
+  void AddFact(Nonnull<const AstNode*> node, const FormedState state) {
+    CARBON_CHECK(!facts_.count(node));
```

Comment:
```suggestion
    CARBON_CHECK(facts_.find(node) == facts_.end());
```

Let's avoid `.count`, I'm not really comfortable with the integer comparisons.